_posts/2017-11-06-cockpit-155.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cockpit 155
+author: pitti
+date: 2017-11-06 17:00
+tags: cockpit linux
+slug: cockpit-155
+category: release
+summary: Cockpit with NFS client support
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 155.
+
+### NFS client support
+
+The Storage page now shows currently mounted NFS volumes and offers to
+mount/unmount them.
```

Comment:
Worth saying that authentication/kerberos is not supported yet?

Reply:
Yes, that can't hurt. Pushed.